{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/api/views.py

```diff
@@ -1,13 +1,12 @@
-from django.contrib.auth import get_user_model
 from rest_framework import status
 from rest_framework.decorators import action
 from rest_framework.mixins import ListModelMixin, RetrieveModelMixin, UpdateModelMixin
 from rest_framework.response import Response
 from rest_framework.viewsets import GenericViewSet
 
-from .serializers import UserSerializer
+from {{ cookiecutter.project_slug }}.users.models import User
 
-User = get_user_model()
+from .serializers import UserSerializer
```

Comment:
You use a local import here and nowhere else; might I suggest the entire `User` app be changed to use local imports? I personally think it's cleaner, and it can encourage new developers to encapsulate their apps and make them more reusable (which I know is the opposite of what this pull request is doing, but still)

Reply:
Ok, I'll open a PR as soon as this one is merged